programs/vote/src/vote_state/mod.rs

```diff
@@ -1985,9 +1999,17 @@ mod tests {
                 )
             });
 
-            let versioned = VoteStateVersions::Current(Box::new(vote_state.take().unwrap()));
+            let versioned = VoteStateVersions::new_current(vote_state.take().unwrap());
             VoteState::serialize(&versioned, &mut max_sized_data).unwrap();
             vote_state = Some(versioned.convert_to_current());
         }
     }
+
+    #[test]
+    fn test_default_vote_state_is_uninitialized() {
+        // The default `VoteState` is stored to de-initialize a zero-balance vote account,
+        // so must remain such that `VoteStateVersions::is_uninitialied()` returns true
```

Comment:
```suggestion
        // so must remain such that `VoteStateVersions::is_uninitialized()` returns true
```

Reply:
Thanks!  I'll squash this in with the rebase atop #14232